instrumentation/net/http/otelhttp/handler.go

```diff
@@ -45,6 +45,7 @@ type Handler struct {
 	meter             metric.Meter
 	propagators       propagation.TextMapPropagator
 	spanStartOptions  []trace.SpanStartOption
+	metricAttributes  []func(string, *http.Request, int) []attribute.KeyValue
```

Comment:
```suggestion
	metricAttributeFns  []func(string, *http.Request, int) []attribute.KeyValue
```

Same here with the name.